exercises/space-age/tests/space-age.rs

```diff
@@ -0,0 +1,69 @@
+extern crate space_age;
+
+use space_age::*;
+
+fn assert_in_delta(expected: f64, actual: f64) {
+    let diff: f64 = expected - actual.abs();
+    let delta: f64 = 0.01;
+    if diff > delta {
+        panic!("Your result of {} should be within {} of the expected result {}",
+               actual,
+               delta,
+               expected)
+    }
+}
+
+#[test]
+fn earth_age() {
+    let duration = Duration::from(1_000_000_000);
+    assert_in_delta(31.69, Earth.years_during(&duration));
```

Comment:
interesting point that `Earth.years_during` is used rather than `Earth::years_during`. This is possible because `Earth` is a struct with no members (a [unit-like struct](https://doc.rust-lang.org/book/structs.html#unit-like-structs) in the nomenclature). Doesn't seem to me like these functions need to take a `self` though...